xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -926,11 +926,13 @@ void CRenderManager::UpdateDisplayLatency()
   float refresh = fps;
   if (g_graphicsContext.GetVideoResolution() == RES_WINDOW)
     refresh = 0; // No idea about refresh rate when windowed, just get the default latency
-  m_displayLatency = (double) g_advancedSettings.GetDisplayLatency(refresh);
-
-  int buffers = g_Windowing.NoOfBuffers();
-  m_displayLatency += (buffers - 1) / fps;
+  m_displayLatency = g_advancedSettings.GetDisplayLatency(refresh);
```

Comment:
the latter seems to be a good idea but maybe we should introduce a new variable for the advanced setting because it's not really displaylatency but some tweak value nodoy has figured out so far why it's needed on some systems and not on others.